examples/sequence-to-sequence/seq2seq_translator/README.md

```diff
@@ -0,0 +1,268 @@
+
+```python
+class EncoderRNN(object):
+
+    def __init__(self, in_vocab, hidden_dim, model):
+        self.in_vocab = in_vocab
+        self.hidden_dim = hidden_dim
+        self.embedding_enc = model.add_lookup_parameters((self.in_vocab, self.hidden_dim))
+        self.rnn_enc = dy.GRUBuilder(1, self.hidden_dim, self.hidden_dim, model)
+
+    def __call__(self, input, hidden):
+        input_embed = dy.lookup(self.embedding_enc, input)
+        state_enc = self.rnn_enc.initial_state(vecs=hidden)
+        state_enc = state_enc.add_input(input_embed)
+        return state_enc.output(), state_enc.h()
+
+    def initHidden(self):
+        return [dy.zeros(self.hidden_dim)] 
+   
+DROPOUT_RATE = 0.1
+
+class AttnDecoderRNN(object):
+
+    def __init__(self, hidden_dim, out_vocab, model, max_length=MAX_LENGTH):
+        self.hidden_dim = hidden_dim
+        self.out_vocab = out_vocab
+        self.max_length = max_length
+        self.embedding_dec = model.add_lookup_parameters((self.out_vocab, self.hidden_dim))
+        self.w_attn = model.add_parameters((self.max_length, self.hidden_dim * 2))
+        self.b_attn = model.add_parameters((self.max_length,))
+        self.w_attn_combine = model.add_parameters((self.hidden_dim, self.hidden_dim * 2))
+        self.b_attn_combine = model.add_parameters((self.hidden_dim,))
+        self.rnn_dec = dy.GRUBuilder(1, self.hidden_dim, self.hidden_dim, model)
+        self.w_dec = model.add_parameters((self.out_vocab, self.hidden_dim))
+        self.b_dec = model.add_parameters((self.out_vocab,))
+
+    def __call__(self, input, hidden, encoder_outptus, dropout=False):
+        input_embed = dy.lookup(self.embedding_dec, input)
+        if dropout:
+            input_embed = dy.dropout(input_embed, DROPOUT_RATE)
+        input_cat = dy.concatenate([input_embed, hidden[0]])
+        w_attn = dy.parameter(self.w_attn)
+        b_attn = dy.parameter(self.b_attn)
+        attn_weights = dy.softmax(w_attn * input_cat + b_attn)
+        attn_applied = encoder_outptus * attn_weights
+        output = dy.concatenate([input_embed, attn_applied])
+        w_attn_combine = dy.parameter(self.w_attn_combine)
+        b_attn_combine = dy.parameter(self.b_attn_combine)
+        output = w_attn_combine * output + b_attn_combine
+        output = dy.rectify(output)
+        state_dec = self.rnn_dec.initial_state(vecs=hidden)
+        state_dec = state_dec.add_input(output)
+        w_dec = dy.parameter(self.w_dec)
+        b_dec = dy.parameter(self.b_dec)
+        output = state_dec.output()
+        output = dy.softmax(w_dec * output + b_dec)
+        return output, state_dec.h(), attn_weights
+
+    def initHidden(self):
+        return [dy.zeros(self.hidden_dim)]
+```
+
+Install the GPU version of Dynet according to the instructions on the [official website](http://dynet.readthedocs.io/en/latest/python.html#installing-a-cutting-edge-and-or-gpu-version).
+
+Then, run the training:
+
+<pre>
```

Comment:
Instead of using `<pre>` tags, can you just use 4 spaces of indent?

Reply:
Modified.